### ISSUE TITLE
Login API is not working

### ISSUE THREAD

**Issue by sentanos** [sentanos/roblox-js#18]
This is ROBLOX's problem once again. They better fix it soon because I would not be thrilled to change the APIs for a second time.

**Comment by sentanos** [comment]
Until ROBLOX fixes their API this alternate login can be used: https://gist.github.com/sentanos/09e7cad77c62fc71b0b5b37d08d58d1e
Replace the login.js file in roblox-js/lib/util with that and users can be logged in. Note that it will not show errors correctly or return the userId and should only be used temporarily until ROBLOX fixes their existing API.

**Comment by Conmmander** [comment]
Hey, just wanted to say, I tried updating my code with this, and it did not fix it.

**Comment by sentanos** [comment]
It works for me and others, make sure the fix was actually applied (make sure the server isn't reinstalling dependencies or not using the new one).

**Comment by Conmmander** [comment]
I'm not sure how I make sure that the server is not reinstalling dependencies, but I am positive I updated the code. I'll look into this, however and see what's going on.

**Comment by The-Tarkol** [comment]
I replaced the file like you said, and this time I get a different error. Did I mess something up?
The error says "Cannot GET /" when I try to navigate to the app. In game it just says 404 not found.

**Comment by sentanos** [comment]
"Cannot GET" is normal for a working app. I'm not sure why you are getting a 404 when using it, though, are you sure you pushed the updates correctly?

**Comment by The-Tarkol** [comment]
I believe so. Let me try again and I'll let you know.

**Comment by The-Tarkol** [comment]
It works now. For some reason I had a / at the end of the url in the script, removing it fixed it. Thank you!

**Comment by Conmmander** [comment]
Nope, I tried reinstalling it, and whatnot. but still nothing changed.

**Comment by sentanos** [comment]
I am pushing updates now, if you are using v3 you can upgrade to the latest version 3.0.2 with `npm install roblox-js`. If you are still using v2 you can install a patch with `npm install roblox-js@2.3.4`. If you are using roblox-js-server please follow the instructions for [updating dependencies](https://github.com/sentanos/roblox-js-server#updating).
